pkg/integrations/v4/logs/cfg_template.go

```diff
@@ -152,3 +152,29 @@ var fbLuaScriptFormat = `function {{ .FnName }}(tag, timestamp, record)
     -- If there is not any matching conditions discard everything
     return -1, 0, 0
  end`
+
+var tooManyFilesWarnMsg = `
+The amount of open files targeted by your Log Forwarding configuration
+files ({{ .TotalFiles }}) exceeds the recommended maximum (1024). The Operating System
```

Comment:
Is there a way we can also pass the hardcoded maximum (1024) using the `fbFileWatchLimit` variable?

Reply:
Sure. I have added a field in the struct to avoid the hard coded recommended maximum.